app/app.rb

```diff
@@ -121,10 +121,11 @@
 
 post '/user/routes' do
   user = user_from_token
-  route = Route.create!(user: , commitment: user.current_commitment, started_at: Time.current)
 
-  { data: route.to_json }
+  route = Route.create!(user: , commitment: user.current_commitment, started_at: Time.current)
 
+  content_type :json
+  route.to_json
```

Comment:
When I change this to an object with data: I get a failure because it is expecting it not to be encapsulated in the data: attribute. I'm using the data_object_for method in my tests. I've also tried using the data_object_for method with no luck. Is there a different methodI should be using?

I may be using this data_object_for thing incorrectly

Reply:
That method just turns a record into a hash. You can do something like this: https://github.com/ittullos/pray4tn/pull/4/commits/83f928e811a399f976c0eda56ebe841bbcf52b5a

Reply:
IMO, one isn't necessarily better than the other, I was just hoping to make it consistent for you on the front end, that's the most important thing. 
Maybe it makes more sense to omit the "data" key, since the majority of the routes don't have it. I'm cool with either one, we should just pick an approach and be consistent so you don't have to handle both versions in the FE.

Reply:
Ahh perfect!! Thanks dude!